quantum/process_keycode/process_steno.c

```diff
@@ -0,0 +1,151 @@
+ #include "process_steno.h"
```

Comment:
Would you mind adding the appropriate copyright headers to this and the .h if you're the sole author?